apps/web/src/components/utils/Notifications.tsx

```diff
@@ -0,0 +1,36 @@
+import { showNotification } from '@mantine/notifications';
+import { Check } from '../../design-system/icons/general/Check';
+import { ErrorIcon } from '../../design-system/icons';
+import { colors } from '../../design-system';
+
+export function successMessage(message: string) {
+  showNotification({
+    message,
+    icon: <Check />,
+    styles: (theme) => ({
+      icon: {
+        width: '22px',
+        height: '22px',
+        marginRight: '10px',
+        color: theme.colorScheme === 'dark' ? `${colors.B15} !important` : `${colors.white} !important`,
+        backgroundColor: colors.success,
+      },
+    }),
+  });
+}
+
+export function errorMessage(message: string) {
+  showNotification({
+    message,
+    icon: <ErrorIcon />,
+    styles: (theme) => ({
+      icon: {
+        width: '22px',
+        height: '22px',
+        marginRight: '10px',
+        color: `${colors.error} !important`,
+        backgroundColor: theme.colorScheme === 'dark' ? colors.B15 : colors.white,
```

Comment:
Shouldn't they be switched? As backgroundColor is the circle (colors.error ) and color is the fill of the SVG.

Reply:
I was thinking the same thing. But I've tested it both ways and it appears the way I had is the way it needs to be.